apps/beacon/main.cpp

```diff
@@ -115,6 +114,7 @@ int main()
   }
   std::this_thread::sleep_for(std::chrono::milliseconds(500));
 
+#if 0
```

Comment:
@n-hutton lets have a chat about this tomorrow

Reply:
I'd like to be in that discussion.